src/openforms/submissions/api/serializers.py

```diff
@@ -346,26 +350,20 @@ def notify_suspension(self, instance: Submission, email: str):
             "save_date": timezone.now(),
             "expiration_date": datetime_removed,
             "continue_url": continue_url,
+            "rendering_text": True,
         }
 
-        html_content = render_to_string(
-            "emails/save_form/save_form.html", context=context
-        )
-        context["rendering_text"] = True
-        text_content = render_to_string(
-            "emails/save_form/save_form.txt", context=context
-        )
-        subject_content = render_to_string(
-            "emails/save_form/save_form_subject.txt", context=context
-        )
-
         send_mail_html(
-            subject_content.strip(),
-            html_content,
+            render_email_template(config.save_form_email_subject, context),
+            render_email_template(config.save_form_email_content, context),
             settings.DEFAULT_FROM_EMAIL,
             [email],
-            fail_silently=False,
-            text_message=text_content,
+            text_message=strip_tags_plus(
+                render_email_template(
+                    config.save_form_email_content, context, rendering_text=True
+                ),
+                keep_leading_whitespace=True,
+            ),
```

Comment:
we don't have any support for `rendering_text` context in the default templates, so you might as well let `send_mail_html` automatically handle the plain text version.